curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/643c95001ff61f0b3c39ce71.md

```diff
@@ -0,0 +1,178 @@
+---
+id: 643c95001ff61f0b3c39ce71
+title: Step 32
+challengeType: 0
+dashedName: step-32
+---
+
+# --description--
+
+The `item` parameter is the current element being processed in the array of the `forEach` loop. You can use it to access the properties of the object. But its better to destructure the object, like this:
+
+```js
+const { title, author, created_at, posts_count, views } = item;
```

Comment:
Let's remove the example here too because campers will be comfortable with `destructuring ` by the time this project comes around.

Reply:
done